### PR TITLE
Update toggles in Quickview->Administration visually

### PR DESCRIPTION
**Pull request by mleeBoeing**
The toggles for Preferences now display ON/OFF.  The toggles in the Administration tab for an App should follow the same design.

### CODE REVIEW

**Comment by blynch11p** [comment]
Those buttons are huge.  Recommend making them a little closer to the size they originally were.
screenshot from 2017-07-20 17-50-11
screenshot from 2017-07-20 17-48-32

File: app/js/components/createEdit/form/Toggle.jsx [hunk 1]
```diff
@@ -14,21 +14,19 @@ var Toggle = React.createClass({
       if (!this.state.value) {
           fullExplanation = isEnabled + this.props.explanation[0];
       }
+    //   console.log(this.props.id1);
```
Comment (blynch11p) [inline app/js/components/createEdit/form/Toggle.jsx]: Unneeded

**Comment by blynch11p** [comment]
:-1: 

The 'singleton' toggle on Edit Listing doesn't not actually save state.  Private Listing appears to however.
There's also one commented out console.log in the code.

**Comment by mleeBoeing** [comment]
Fixed

**Comment by blynch11p** [comment]
Corrections fixed the issues with Singleton and side-effects I was seeing with the Admin tab. :+1:

**Pull request merged**